src/main/java/oc/wh40k/units/im/IMBattleSistersSquad.java

```diff
@@ -2,62 +2,65 @@
 
 import oc.*;
 
+import static oc.KeyWord.*;
+
 public class IMBattleSistersSquad extends Eintrag {
 
     AnzahlPanel squad;
-    OptionsUpgradeGruppe o1;
-    OptionsUpgradeGruppe o2;
-    RuestkammerStarter rkBoss;
-    RuestkammerStarter rkTransport;
+    OptionsZaehlerGruppe bolters;
```

Comment:
New fields should `final` if possible and have the most restrictive access modifier if possible. So `private` in this case.